environments.json

```diff
@@ -3694,12 +3694,25 @@
     "short": "JrS 2.2.0",
     "platformtype": "engine",
     "release": "2020-02-11",
-    "obsolete": false,
+    "obsolete": true,
     "test_suites": [
       "es5",
       "es6"
     ]
   },
+  "jerryscript2_3_0": {
+    "full": "JerryScript 2.3.0",
+    "family": "JerryScript",
+    "short": "JrS 2.3.0",
+    "platformtype": "engine",
+    "release": "2020-06-12",
+    "obsolete": false,
+    "test_suites": [
+      "es5",
+      "es6",
+      "es2016plus"
+    ]
+  },
```

Comment:
```suggestion
  "jerryscript2_3_0": {
    "full": "JerryScript 2.3.0",
    "family": "JerryScript",
    "short": "JrS 2.3.0",
    "platformtype": "engine",
    "release": "2020-06-12",
    "equals": "jerryscript2_2_0",
    "obsolete": false,
    "test_suites": [
      "es5",
      "es6",
      "es2016plus"
    ]
  },
```

Reply:
It is inherited properly without this change.